### PR TITLE
[Feature][UI Next] Add file manage

### PR DESCRIPTION
**Pull request by devosend**
## Purpose of the pull request
#7785 

## Brief change log
- [x] File Manage

## display
- file list 
image

- create folder 
image
- create file
image

- edit file
image

### CODE REVIEW

**Review by labbomb — COMMENTED** [review]
@songjianet Please review it

File: dolphinscheduler-ui-next/src/locales/modules/en_US.ts [hunk 1]
```diff
@@ -151,6 +151,42 @@ const monitor = {
   },
 }
 
+const resource = {
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/locales/modules/en_US.ts:154]: resource = {
  file: {
    file_manage: 'File Manage',
    ....
  }
}

File: dolphinscheduler-ui-next/src/service/service.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ import axios, {
   AxiosRequestHeaders,
 } from 'axios'
 import qs from 'qs'
+import _ from 'lodash'
+import $ from 'jquery'
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/service/service.ts]: To download a file, you can get the corresponding `url` from the corresponding interception in `axios`, and then create a form for downloading. It is not recommended to use `jquery` for this.

File: dolphinscheduler-ui-next/src/views/resource/file/edit/resource-file-edit.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                  width: '90%',
+                }}
+              >
+                <MonacoEditor
+                  ref='codeEditorRef'
+                  modelValue={this.resourceViewRef?.value.content}
+                />
+              </div>
+            </NFormItem>
+            {this.routeNameRef === 'resource-file-edit' ? (
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/views/resource/file/edit/resource-file-edit.tsx]: this.routeNameRef === 'resource-file-edit' && 
This is fine. It seems that there is no need to use ternary operations in this place.

File: dolphinscheduler-ui-next/src/views/resource/file/table/table-action.tsx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  emits: ['updateList', 'renameResource'],
+  setup(props, { emit }) {
+    const { t } = useI18n()
+    const router: Router = useRouter()
+
+    const rtDisb: IRtDisb = (name, size) => {
+      const i = name.lastIndexOf('.')
+      const a = name.substring(i, name.length)
+      let flag = _.includes(fileTypeArr, _.trimStart(a, '.'))
+      if (flag && size < 1000000) {
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/views/resource/file/table/table-action.tsx]: return !flag && size < 1000000

File: dolphinscheduler-ui-next/src/views/resource/file/table/use-table.ts [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useRouter } from 'vue-router'
+
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/views/resource/file/table/use-table.ts]: delete entry

**Review by songjianet — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui-next/src/views/resource/file/use-file.ts [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    return state
+  }
+
+  const getResourceView = (id: number) => {
+    const params = {
+      skipLineNum: 0,
+      limit: 3000,
+    }
+    const { state } = useAsyncState(
+      viewResource(params, id).then((res: any) => {
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/views/resource/file/use-file.ts]: viewResource(params, id)

File: dolphinscheduler-ui-next/src/views/resource/file/edit/use-edit.ts [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  const { t } = useI18n()
+  const router: Router = useRouter()
+
+  const getResourceView = (id: number) => {
+    const params = {
+      skipLineNum: 0,
+      limit: 3000,
+    }
+    const { state } = useAsyncState(
+      viewResource(params, id).then((res: any) => {
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/views/resource/file/edit/use-edit.ts]: viewResource(params, id)

**Review by songjianet — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui-next/src/views/resource/file/use-file.ts (hunk 6 above)
Comment (songjianet) [reply dolphinscheduler-ui-next/src/views/resource/file/use-file.ts]: If you don't need to process the success data, you can omit `then`.

File: dolphinscheduler-ui-next/src/views/resource/file/table/table-action.tsx (hunk 4 above)
Comment (devosend) [reply dolphinscheduler-ui-next/src/views/resource/file/table/table-action.tsx]: This should be `return !(flag && size < 1000000)`

**Comment by devosend** [comment]
PTAL @songjianet

**Review by songjianet — APPROVED** [review]
LGTM

**Review by labbomb — APPROVED** [review]
LGTM

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks @devosend

**Pull request merged**